contracts/priv/cloud-priv.yml

```diff
@@ -281,6 +281,123 @@ paths:
+    put:
+      operationId: PutSetupUser
+      tags:
+        - Setup
+      description: 'Create a new user, organization, and bucket. Idempotent. Requests can be retried.'
+      requestBody:
+        description: Create user and org with supplied IDs
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/OnboardingWithIDsRequest'
+      responses:
+        '201':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/OnboardingWithTokenResponse'
+          description: 'The user name and ID, the org name and ID, the token for user if requested'
+        '400':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/responses/ServerError'
+          description: 'Bad request. Data in request is invalid. Examples include Invalid IDs, invalid names, and invalid limits'
+        '401':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/responses/ServerError'
+          description: Credentials not provided
+        '403':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/responses/ServerError'
+          description: Insufficient credentials to create an organization
+        '409':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/responses/ServerError'
+          description: 'Conflict. Causes can be user already exists with a different name, org already exists with a different name, org exists with different limits.'
+        '501':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/responses/ServerError'
+          description: Internal Server Error. Request can be retried
+        default:
+          $ref: '#/components/responses/ServerError'
+          description: Unexpected error
+  /user:
+    put:
+      operationId: PutUserPrivate
+      tags:
+        - Users
+      summary: Create a user. Idempotent
+      parameters:
+        - in: header
+          name: Zap-Trace-Span
+          description: OpenTracing span context
+          example:
+            trace_id: '1'
+            span_id: '1'
+            baggage:
+              key: value
+          required: false
+          schema:
+            type: string
+      requestBody:
+        description: User to create and org id to add user to
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/OnboardingUserRequest'
+      responses:
+        '201':
+          description: User created
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/OnboardingWithIDsResponse'
```

Comment:
Should this be named something like `OnboardingUserResponse` to avoid confusion with the `OnboardingWithIDsRequest` above?

Reply:
We could, although the response is potentially reusable for other APIs. So I name it by what the response contains, not the context of the api from which it is called. if you find it confusing, I can change it

Reply:
That's a good convention, so I'm fine with it. It just felt a bit odd that an `OnboardingWithIDsResponse` didn't go with an `OnboardingWithIDsRequest`. Ultimately, the names aren't going to be used directly in the Quartz code, so whatever makes sense to you is fine.